apps/rush-lib/src/cli/actions/CustomRushAction.ts

```diff
@@ -148,6 +155,13 @@ export class CustomRushAction extends BaseRushAction {
       parameterShortName: '-v',
       description: 'Display the logs during the build, rather than just displaying the build status summary'
     });
+    if (this.options.actionVerb === 'build') {
+      this._ignoreDownstream = this.defineFlagParameter({
+        parameterLongName: '--no-deps',
```

Comment:
The point of the "long name" is that it uses intelligible full English words.  If this is an inner loop scenario, you should probably define a short name.

Also the name should be clearly a modifier for these options:

```
  -t PROJECT1, --to PROJECT1
                        Run command in the specified project and all of its
                        dependencies
  -f PROJECT2, --from PROJECT2
                        Run command in all projects that directly or
                        indirectly depend on the specified project
```

Reply:
Well...  does it affect the "-t" switch?

Reply:
I suggested that flag to Marc. I think it makes more sense.

---
In reply to: [164221768](https://github.com/Microsoft/web-build-tools/pull/514#discussion_r164221768) [](ancestors = 164221768)